python_utils/testRobot.py

```diff
@@ -241,6 +286,27 @@ def createRobotCommand(robot_id, test, tick_counter, period_fraction):
 				basestation.write(cmd_encoded)
 				parser.writeBytes(cmd_encoded)
 				last_robotcommand_time = time.time()
+				
+				# Create and send new PID gains (default 0)
+				setPID = createSetPIDCommand() #put PID gains as arguments to change them
+				setPID_encoded = setPID.encode()
+				basestation.write(setPID_encoded)
```

Comment:
setPIDGains should not be written 60 times per second. Either once, or maybe at most once per second